package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "k6-jslib-aws",
     "repository": "https://github.com/grafana/k6-jslib-aws",
-    "version": "0.7.1",
+    "version": "0.7.2",
```

Comment:
I would prefer if we could drop the version bump from this PR. We, as the maintainers, are responsible for producing and publishing releases, and I think we'll pack a couple more features in the next one 😄 🙇🏻

Reply:
Done.